sim/rogue/items.go

```diff
@@ -85,5 +90,107 @@ func init() {
 		}))
 	})
 
+	// https://www.wowhead.com/classic/item=224122/dream-eater
+	// Damaging finishing moves have a 20% chance per combo point to restore 10 energy.
+	core.NewItemEffect(DreamEater, func(agent core.Agent) {
+		rogue := agent.(RogueAgent).GetRogue()
+
+		cpMetrics := rogue.NewEnergyMetrics(core.ActionID{SpellID: 451439})
+		rogue.OnComboPointsSpent(func(sim *core.Simulation, spell *core.Spell, comboPoints int32) {
+			if sim.Proc(0.2*float64(comboPoints), "Dream Eater") {
+				rogue.AddEnergy(sim, 10, cpMetrics)
+			}
+		})
+	})
+
+	// https://www.wowhead.com/classic/item=231287/renatakis-charm-of-trickery
+	// Use: Instantly increases your energy by 60. If Cutthroat is engraved, gain an activation of Cutthroat's Ambush effect. (2 Min Cooldown)
+	core.NewItemEffect(RenatakisCharmofTrickery, func(agent core.Agent) {
+		rogue := agent.(RogueAgent).GetRogue()
+		cpMetrics := rogue.NewEnergyMetrics(core.ActionID{SpellID: 468458})
+		duration := time.Second * 10
```

Comment:
Since this is only used once and could be confused with the cooldown duration what do you think about just not assigning it to a variable and just using the value below?

Reply:
agreed